src/Tizen.MachineLearning.Nnstreamer/Tizen.MachineLearning.Nnstreamer/Pipeline.cs

```diff
@@ -0,0 +1,90 @@
+﻿/*
+* Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using Log = Tizen.Log;
+
+namespace Tizen.MachineLearning.Nnstreamer
+{
+    /// <summary>
+    /// The Pipeline API provides interfaces to create and execute stream pipelines with neural networks and sensors.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    /// <remarks>
+    /// The Pipeline API creates a stream pipeline with NNStreamer plugins, GStreamer plugins, and sensor input devices.
+    /// It provides the interface to push data to the pipeline from the application get the inference result from the pipeline to the application.
+    /// It also provides the Interfaces to start/stop/destroy the pipeline and other utility functions such as controling switches and valves in the pipeline.
+    /// </remarks>
+    public class Pipeline
+    {
+        static string Tag = "Nnstreamer.Pipeline";
+        private IntPtr _pipelineHandle;
```

Comment:
When use native handle as a member of the class, the Disposable pattern should be used for safety. 
see: https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose